backend/server/core/enums/stateChallenge.ts

```diff
@@ -0,0 +1,5 @@
+export enum StateChallenge {
```

Comment:
Please rename all files in the folder `enum` as `<name>.enum.ts'. And all files in the `interfaces` folder as `<name>.interface.ts`.